iocage_lib/ioc_common.py

```diff
@@ -770,3 +770,33 @@ def match_to_dir(iocroot, uuid, old_uuid=None):
         return matches[0]
     else:
         return None
+
+
+def consume_and_log(exec_gen, return_list=False, callback=None, silent=False):
```

Comment:
Switching return value behavior based on method parameter is usually a bad idea, why we cant just always return list, and whoever is using the method can decide what to do with it?